src/main/java/org/example/entities/Department.java

```diff
@@ -0,0 +1,23 @@
+package org.example.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "skyrius")
+public class Department {
+    @Id
+    @Column(name = "pavadinimas")
+    private String title;
```

Comment:
Rekomenduočiau atskirus klasės properties atskirti naujomis eilutėmis:)

Reply:
mmm, tai tos spaced eilutes ne ne anatocijoms paliktos o tiesiog dėl atskirimo :D O aš vis galvodavau, kodėl automatu kuriant į naują eilute vis numesdavo :D

Reply:
aha, be eilučių labai sulimpa viskas tada :D